.github/workflows/trivy.yaml

```diff
@@ -0,0 +1,105 @@
+name: Security Scans
```

Comment:
Why moving this out of the CI/CD scope? I think this way we won't be able to use them in `needs` sections.